admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/service/impl/ValidDocumentServiceImpl.java

```diff
@@ -235,11 +235,13 @@ public ValidDocCategoryAndDocTypeResponseDto getValidDocumentByLangCode(String l
 	 * int, int, java.lang.String, java.lang.String)
 	 */
 	@Override
-	public PageDto<ValidDocumentExtnDto> getValidDocuments(int pageNumber, int pageSize, String sortBy,
+	public PageDto<ValidDocumentExtnDto> getValidDocuments(int pageNumber, Integer pageSize, String sortBy,
 			String orderBy) {
 		List<ValidDocumentExtnDto> validDocs = null;
 		PageDto<ValidDocumentExtnDto> pageDto = null;
 		try {
+			if(null==pageSize || pageSize==0)
+			 pageSize = documentRepository.getTotalValidDocuments();
```

Comment:
There is a requirement for one of the bug raised by Mayura  mosip 17983 , as per the discussion with Dhanendra added this condition ,in case if page size is not provided then all the records needs to be shown.